.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "IHM-dictionary"]
+	path = IHM-dictionary
+	url = https://github.com/ihmwg/IHM-dictionary
```

Comment:
I guess this will need to be updated for IHMCIF.

Reply:
Yes, it will. Thanks @benmwebb.